### DISCUSSION TITLE
Template questions

### DISCUSSION THREAD

**Discussion by frankjmueller** [wort-schule/wort.schule#506] [General]
@phylor Kann es sein, dass die Stichwörter eines Wortes nicht im Standardtemplate enthalten sind?
![grafik](https://github.com/wort-schule/wort.schule/assets/163656194/c4d77d34-472b-4887-ac83-9022aaac1dd5)

Und: wie kann ich von außen auf ein öffentliches Setting verweisen? Geht das schon?
z.B. für 
https://wort.schule/seite/word_view_settings/1
Da war ja die Idee, dass die über einen Link übernommen werden sollen und per Cookie gespeichert. Da bräuchten wir vermutlich noch eine Auswahlseite Ansicht (die über das Menü unten bei Impressum usw.) erreichbar sein könnte.

Noch eine Frage:
Kann ich in den Templates ein Fallback setzen? Also z.B. wenn das Feld Sprechsilben leer ist (wie bei Libelle), dann zeige das Wort an. (Das unformatierte Wort selbst würde ich sonst nämlich aus manchen Templates rausnehmen, weil es sonst ja doppelt da steht.)

**Comment by phylor** [comment]
> @phylor Kann es sein, dass die Stichwörter eines Wortes nicht im Standardtemplate enthalten sind?

Das ist sehr wahrscheinlich ein Bug. Schau ich mir in https://github.com/wort-schule/wort.schule/issues/507 an.

**Comment by phylor** [comment]
> Und: wie kann ich von außen auf ein öffentliches Setting verweisen? Geht das schon?

Ja, so: https://wort.schule/?word_view_setting_id=1

Wollen wir bei der Ansichteinstellung den Link noch hinzufügen? Dann ist das Feature nicht ganz versteckt :slightly_smiling_face:

**Reply by frankjmueller** [reply]
Ja, bitte. Und ich kann als Admin, die nicht öffentlichen Profile der anderen User sehen. Das ist vermutlich auch nicht so gedacht oder?

**Reply by phylor** [reply]
Doch, Admins sehen alle, auch die privaten. Admins dürfen im System generell alles tun.

Admin ist nach meinem Verständnis aber auch eine Spezialrolle, wovon es nicht so viele geben sollte. Falls ihr dies anders nutzt, müssten wir die Berechtigungen anpassen.

**Reply by frankjmueller** [reply]
Okay. Dann ist es gut so.

**Comment by phylor** [comment]
> Kann ich in den Templates ein Fallback setzen? Also z.B. wenn das Feld Sprechsilben leer ist (wie bei Libelle), dann zeige das Wort an.

Das sollte in Liquid gehen. Aber: die oberste Box kannst du im Moment in Liquid nicht anpassen. Du fügst lediglich `{{ header }}` ein und die gesamte Box erscheint. Die müsste ich somit auch erst in Liquid umschreiben, bevor du's anpassen kannst.

Soll ich das machen? Alternative wäre wir entscheiden uns für eine Lösung (z.B. immer nur die Silben-Darstellung anzeigen ohne den Titel darüber).

**Reply by frankjmueller** [reply]
Lass uns da nochmal überlegen. Ich schau mir das nochmal an.

**Reply by frankjmueller** [reply]
Hmmm. Sollen wir da nochmal drüber sprechen?
Im Moment ist es doppelt gemoppelt. Also beim Eisbär steht einmal der Eisbär und dann in kleiner die Silbenvariante drunter.
![grafik](https://github.com/wort-schule/wort.schule/assets/163656194/748f278d-6c3f-441c-b9a1-e910e5455dbe)

Wenn keine Silben vorhanden sind steht da aber gar nix.
![grafik](https://github.com/wort-schule/wort.schule/assets/163656194/f5be0fab-5d89-4698-b022-590e6b6a62a7)

**Reply by phylor** [reply]
Ja, das stimmt. Wollen wir das obere entfernen und dafür die Spezialschrift grösser darstellen?

Du müsstest mir dann nur noch sagen wo der Artikel hinkommt. Vor oder nach dem Lineaturhäuschen? Auch in der Spezialschrift?

**Reply by frankjmueller** [reply]
Ja, bitte. Und ein Fallback falls keine Silben vorhanden sind. Der Artikel kommt hinter das Häuschen, auch in der Spezialschrift.
